azure/terraform.sh

```diff
@@ -24,6 +24,15 @@ function clean_environment() {
 }
 
 function download_tool() {
+
+  cpu_brand=$(sysctl -n machdep.cpu.brand_string)
```

Comment:
This command is specific to macOS. Therefore, since this script is also used on Windows machines occasionally (by consultants), I suggest wrapping it with a check to ensure it's being run on a MacBook.